Expecto.Tests/StatisticsTests.fs

```diff
@@ -0,0 +1,75 @@
+module Expecto.StatisticsTests
+
+open Expecto
+open System.Collections.Generic
+
+[<Tests>]
+let statisticsTests =
+  ftestList "statistics tests" [
```

Comment:
`ftestList `: Do you want this list to be focused?

Reply:
Thanks. It's WIP and I'll remove later.